dokumentlager/domene/src/main/java/no/nav/foreldrepenger/melding/dokumentdata/DokumentMalType.java

```diff
@@ -50,6 +45,18 @@
     public static final String KLAGE_OVERSENDT_KLAGEINSTANS = "KOVKLA";
     public static final String KLAGE_HJEMSENDT = "KHJEMS";
 
+    // Disse beholdes her grunnet gamle brev i databasen:
```

Comment:
Burde vi slettet disse gamle brevene fra databasen? De skal vel ikke rekjøres?

Reply:
Jeg tror ikke vi bør slette disse brevene spesifikt, men vi kan evt. slette alle data eldre enn feks 1 år, og så gjenta det på et senere tidspunkt, som da også vil treffe resten av disse brevene. Tenker vi minimum bør ha data fra siste året i tilfelle behov for feilsøking?